documentation-website/src/components/t.ts

```diff
@@ -12,8 +12,15 @@ export default async(lnkey: languageKey, global?: object,): Promise<string> => {
     return '';
   }
   const that: Window = (global ?? window) as Window;
-  const language = (that?.Navigator?.language ?? 'en')
-    .replace(/-.*$/u, '',);
+  /*const language = (
+    (that?.localStorage?.getItem('language',) || 'en')
+    ?? that?.Navigator?.language
+    ?? 'en'
+  ).replace(/-.*$/u, '',);*/
+  let language = (
```

Comment:
the above looks better, just replace the replace with split :)

Reply:
Fixed in my latest commit (I don't really know how to fix the "conflibot / conflibot (pull_request)" failed test though 😅